.github/workflows/next.yml

```diff
@@ -56,8 +56,7 @@ jobs:
           composer config preferred-install dist
           composer config repositories.0 path $GITHUB_WORKSPACE
           composer config repositories.1 composer https://packages.drupal.org/8
-          composer config allow-plugins true -n
```

Comment:
By removing the `allow-plugins` command, this could cause the CI to hang at a prompt, right?

Reply:
Looking at a CI build log, it doesn't look like there are any plugins. I suppose there could be some dependency in the future that requires a plugin that would need to be allowed to run? I guess the composer `--no-interaction` flag would keep the build from hanging. But then the plugin wouldn't run...

Reply:
Ah! It prevents a command prompt when installing a Composer plugin in the CI. I didn't realize that that is what the purpose of the command is. I've removed the removal.